Index: ../server/app/api/users.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from flask import Blueprint, request, jsonify\r\nfrom app.db import test_db_connection, get_listings_by_name\r\n\r\nfrom flask_cors import CORS\r\nfrom app.api.utils import expect\r\nfrom datetime import datetime\r\n\r\nusers_api_v1 = Blueprint(\r\n    'users_api_v1', 'users_api_v1', url_prefix='/api/v1/users')\r\n\r\nCORS(users_api_v1)\r\n\r\n\r\n@users_api_v1.route('/', methods=['GET'])\r\ndef api_get_movies():\r\n    response = \"Welcome to the API!\"\r\n    return jsonify(response)\r\n\r\n@users_api_v1.route('/test', methods=['GET'])\r\ndef api_test_db():\r\n    response = test_db_connection()\r\n    return jsonify(response)\r\n\r\n# TODO: Implement the /listings route to get listings from the database.\r\n@users_api_v1.route('/listings', methods=['GET'])\r\ndef api_get_listings():\r\n    name = request.args.get('name', default=None, type=str)\r\n    listings = get_listings_by_name(name)\r\n    return jsonify(listings)
===================================================================
diff --git a/../server/app/api/users.py b/../server/app/api/users.py
--- a/../server/app/api/users.py	
+++ b/../server/app/api/users.py	
@@ -16,11 +16,13 @@
     response = "Welcome to the API!"
     return jsonify(response)
 
+
 @users_api_v1.route('/test', methods=['GET'])
 def api_test_db():
     response = test_db_connection()
     return jsonify(response)
 
+
 # TODO: Implement the /listings route to get listings from the database.
 @users_api_v1.route('/listings', methods=['GET'])
 def api_get_listings():
